ddl/index.go

```diff
@@ -928,7 +929,7 @@ func (w *worker) buildIndexForReorgInfo(t table.Table, workers []*addIndexWorker
 
 	startHandle, endHandle := reorgInfo.StartHandle, reorgInfo.EndHandle
 	for {
-		kvRanges, err := splitTableRanges(t, reorgInfo.d.store, startHandle, endHandle, reorgInfo.PartitionID)
+		kvRanges, err := splitTableRanges(t, reorgInfo.d.store, startHandle, endHandle)
```

Comment:
Is the table t a `Partition` or a `Table` or a `PartitionedTable` ?
Note that `spliteTableRange` can't work on `PartitionedTable`.

Reply:
Why splitTableRange can't work? The PartitionedTable is inited by `initTableCommonWithIndices`, so the id is the partitionID.

Reply:
PartitionedTable is not a partition.